tensorflow/lite/micro/compression/BUILD

```diff
@@ -97,6 +97,43 @@ py_test(
     ],
 )
 
+py_binary(
+    name = "compress",
+    srcs = [
+        "compress.py",
+    ],
+    deps = [
+        ":metadata_py",
+        ":model_facade",
+        ":spec",
+        "@absl_py//absl:app",
+        "@absl_py//absl/flags",
+        "@flatbuffers//:runtime_py",
+        requirement("bitarray"),
+        requirement("flatbuffers"),
```

Comment:
this duplicate with the above flatbuffer include? Prefer to keep the top one for easier sync for me.